cartography/data/indexes.cypher

```diff
@@ -33,6 +33,7 @@ CREATE INDEX ON :EC2Instance(id);
 CREATE INDEX ON :EC2Instance(instanceid);
 CREATE INDEX ON :EC2Instance(publicdnsname);
 CREATE INDEX ON :EC2KeyPair(id);
+CREATE INDEX ON :EC2PrivateIp(id);
```

Comment:
This was it!!! 

No index = 3 days (or not even finishing). Index = 30 seconds.

Reply:
nice